src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,61 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final DateTimeFormatter DATE_FORMATE = DateTimeFormatter.ofPattern("dd.MM.yyyy");
+    private static final String DASH = " - ";
+    private static final int INDEX_ONE = 1;
```

Comment:
Does these indexes have some meaning? Name INDEX_ONE is almost the same as magic number 1, there is no additional context in you names

Reply:
Change names of variables, thanks.